daemon/info.go

```diff
@@ -67,6 +67,17 @@ func (daemon *Daemon) SystemInfo() (*types.Info, error) {
 		}
 	})
 
+	var securityOptions []string
+	if sysInfo.AppArmor {
+		securityOptions = append(securityOptions, "apparmor")
+	}
+	if sysInfo.Seccomp {
+		securityOptions = append(securityOptions, "seccomp")
```

Comment:
i.e. we have a bug report mentioning it's using seccomp but we don't have the profile (if it isn't the default)